style.css

```diff
@@ -0,0 +1,66 @@
+.wrapper{
+    width: 1020px;
+    margin: 0 auto;
+}
+
+.keyboard{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    border: 2px solid black;
+    padding: 5px;
+    width: 100%;
+    height: auto;
+    margin-bottom: 10px;
+}
+
+.button{
+   width: 50px;
+   height: 50px;
+   border: 1px solid black;
+   margin: 3px;
+   padding: 3px;
+   float: left;
+}
+
+.textarea{
+    margin-top: 40px;
+    width: 100%;
+    height: 200px;
+    padding: 5px;
+    font-size: 15px;
+    resize: none;
+    border: 2px solid black;
+}
+
+.active{
+    background-color:  orange;
+}
+.clearfix{
+    display: block;
+    width: 100%;
+    clear: both;
+}
+
+.backspace,
+.caps{
+    width: 120px;
+}
+
+.space {
```

Comment:
Make css looks great. Remove unnecessary indent like and add space between selectors and brackets `.caps {`

Reply:
fix